nylas/client.py

```diff
@@ -52,6 +53,16 @@ def applications(self) -> Applications:
         """
         return Applications(self.http_client)
 
+    @property
+    def credentials(self) -> Credentials:
+        """
+        Access the Credentials API.
+
+        Returns:
+            The Credentials API.
+        """
+        return Credentials(self.http_client)
+
```

Comment:
Hey @mrashed-dev 

This is a good point and I have another thought. Since this endpoint has a prefix for connectors(integrations), I was thinking it might be better included it there. However, I didn't see an integrations resource, so do you think it's better to add it to auth?

Then again I see that the credentials is mainly used for authentication, so for the time being I will add it there.